webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -290,6 +290,57 @@
 
                   knora-base:hasDefaultDeletePermission knora-base:Owner .
 
+    ### ###########################################
+    ### beol:hasSon
+
+    :hasSon rdf:type owl:ObjectProperty ;
+
+                      rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+                      rdfs:label "hat Sohn"@de ,
+                                 "a fils"@fr ,
+                                 "ha figlio"@it ,
+                                 "has son"@en ;
+
+                      rdfs:comment """Repräsentiert eine Vater-Sohn Beziehung"""@de ;
+
+                      knora-base:subjectClassConstraint :person ;
+
+                      knora-base:objectClassConstraint :person ;
+
+                      salsah-gui:guiOrder "5"^^xsd:integer ;
+
+                      salsah-gui:guiElement salsah-gui:Searchbox ;
+
+                      knora-base:hasDefaultViewPermission knora-base:UnknownUser ,
+                                                          knora-base:KnownUser ;
+
+                      knora-base:hasDefaultModifyPermission knora-base:ProjectMember ;
+
+                      knora-base:hasDefaultDeletePermission knora-base:Owner .
+
+    ### ###########################################
+    ### beol:hasSonValue
+
+    :hasSonValue rdf:type owl:ObjectProperty ;
+
+                    rdfs:subPropertyOf knora-base:hasLinkToValue ;
+
+                    knora-base:subjectClassConstraint :person ;
```

Comment:
Maybe we should make this a `knora-base:Person` so we could link to persons outside the BEOL project (yes, I added person ;-))